### PR TITLE
AB#98132 AB#98131 AB#98129 Update Application splash screen

### PR DESCRIPTION
**Pull request by dawid-pawulski**
### 🛠 Changes being made

- update application start view
- add start view when starting project from frontdoor
- add creating ahp project from front door project
- add marking frontdoor project as used when ahp project was created

### 📸 Screenshots (optional)

image

image

### 🏎 Quality check

- [x] Have you performed local tests?
- [x] Are integration tests passing locally?
- [x] Have you checked WCAG (included screenshot)

### CODE REVIEW

**Review by Carq — APPROVED** [review]

File: src/HE.Investment.AHP.WWW/Views/Project/Start.cshtml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+@using HE.Investment.AHP.WWW.Views.Application;
+@using HE.Investment.AHP.WWW.Config;
+@using HE.Investment.AHP.WWW.Models.Project;
+@inject IAhpExternalLinks ExternalLinks
+
+@model ProjectBasicModel;
+@{
+    var title = ApplicationPageTitles.Start(Model.SelectedProgramme.FullName, Model.SelectedProgramme.ShortName);
```
Comment (Carq) [inline src/HE.Investment.AHP.WWW/Views/Project/Start.cshtml:8]: Instead of ` ApplicationPageTitles` use `ProjectPageTitles`

**Review by Carq — APPROVED** [review]

File: src/HE.Investment.AHP.Domain/Project/EventHandlers/MarkFrontDoorProjectAsCompletedEventHandler.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    private readonly IPrefillDataRepository _repository;
+
+    public MarkFrontDoorProjectAsCompletedEventHandler(IPrefillDataRepository repository)
+    {
+        _repository = repository;
+    }
+
+    public async Task Handle(AhpProjectHasBeenCreatedEvent domainEvent, CancellationToken cancellationToken)
+    {
+        await _repository.MarkProjectAsUsed(new FrontDoorProjectId(domainEvent.FrontDoorProjectId.ToGuidAsString()), cancellationToken);
```
Comment (jjankowski87) [inline src/HE.Investment.AHP.Domain/Project/EventHandlers/MarkFrontDoorProjectAsCompletedEventHandler.cs:19]: why just don't pass `domainEvent.FrontDoorProjectId`? It's the same type

File: src/HE.Investment.AHP.Domain/Project/Repositories/ProjectRepository.cs [hunk 3]
```diff
@@ -93,6 +99,22 @@ public async Task<AhpProjectSites> GetProjectSites(AhpProjectId id, AhpUserAccou
             projects.totalItemsCount);
     }
 
+    public async Task<AhpProjectId> CreateProject(ProjectPrefillData frontDoorProject, AhpUserAccount userAccount, CancellationToken cancellationToken)
+    {
+        var projectId = await _projectCrmContext.CreateProject(
+            userAccount.UserGlobalId.ToString(),
+            userAccount.SelectedOrganisationId().ToGuidAsString(),
+            userAccount.Consortium.ConsortiumId.ToString(),
```
Comment (jjankowski87) [inline src/HE.Investment.AHP.Domain/Project/Repositories/ProjectRepository.cs:107]: Maybe use ToGuidAsString for ConsortiumId as well? Are you sure it's not short guid?

File: src/HE.Investment.AHP.WWW/Controllers/ProjectController.cs [hunk 4]
```diff
@@ -15,9 +19,44 @@ public class ProjectController : Controller
...
+    public async Task<IActionResult> Start([FromQuery] string fdProjectId, CancellationToken cancellationToken)
+    {
+        var userAccount = await _ahpUserContext.GetSelectedAccount();
+        if (userAccount.Consortium.HasNoConsortium || await _ahpAccessContext.IsConsortiumLeadPartner())
+        {
+            var availableProgrammes = await _mediator.Send(new GetAvailableProgrammesQuery(), cancellationToken);
+            return View(new ProjectBasicModel(fdProjectId, availableProgrammes[0]));
+        }
+
+        return RedirectToAction("ContactHomesEngland", "ConsortiumMember", new { consortiumId = userAccount.Consortium.ConsortiumId });
```
Comment (jjankowski87) [inline src/HE.Investment.AHP.WWW/Controllers/ProjectController.cs:44]: Does it work when value object is passed as route parameter? Probably `ConsortiumId.Value` needed

**Review by jjankowski87 — APPROVED** [review]

**Pull request merged**